tests/soc_descs/blackhole_140_arch.yaml

```diff
@@ -0,0 +1,72 @@
+# Note taken from software repo - may need updates.
+grid:
```

Comment:
are we not denoting the SiFi riscv cores in the soc desc?

Reply:
not at the moment.
SOC descriptor parser doesn't need another category so let's add those when they are ready to be used.